docs/reference/commandline/plugin_create.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "plugin create"
+description: "the plugin create command description and usage"
+keywords: ["plugin, create"]
+advisory: "experimental"
+---
+
+<!-- This file is maintained within the docker/docker Github
+     repository at https://github.com/docker/docker/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# plugin create (experimental)
+
+```markdown
+Usage:  docker plugin create [OPTIONS] reponame[:tag] PATH-TO-ROOFS
+
+create a plugin from the given PATH-TO-ROOFS, which contains the plugin's root filesystem and the manifest file, manifest.json
+
+Options:
+      --compress   Compress the context using gzip 
+      --help       Print usage
+```
+
+Creates a plugin. Before creating the plugin, prepare the plugin's root filesystem as well as
+the manifest.json (https://github.com/docker/docker/blob/master/docs/extend/manifest.md)
```

Comment:
Should `manifest` be renamed? https://github.com/docker/docker/pull/28148

@vieux ^^